### PR TITLE
"Ignore white space" in config

### PR DESCRIPTION
**Pull request by ZimGil**
## "Ignore white space" in config

  * Showing or Ignoring white space diff now configurable
    > Default is `false`.
    > User can configure otherwise in [configuration file](https://github.com/FredrikNoren/ungit#configuring)

### CODE REVIEW

**Comment by jung-kim** [comment]
Thank!

Can I ask to modify version number in `package.json` and `package-lock.json` via `npm i` and edit readme.md to reflect the change?

**Comment by ZimGil** [comment]
@jung-kim  I assume you by `readme.md` you ment `changelog.md`?

File: source/config.js [hunk 1]
```diff
@@ -214,6 +217,7 @@ let argv = yargs
 .describe('numberOfNodesPerLoad', 'number of nodes to load for each git.log call')
 .describe('mergeTool', 'the git merge tool to use when resolving conflicts')
 .describe('diffType', 'Prefered default diff type used. Can be `"textdiff"` or `"sidebysidediff"`.')
+.describe('ignoreWhiteSpaceDiff', 'Specify whether to Ignore or Show white space diff')
```
Comment (jung-kim) [inline source/config.js]: You would also want to add to help parser 
```js
.boolean('ignoreWhiteSpaceDiff')
```

**Comment by jung-kim** [comment]
yes you are correct.

Also, I do apologize for forgetting one more thing. and after that I will merge it.  Thanks,

**Pull request merged**